bin/infection

```diff
@@ -48,6 +48,13 @@ if (\in_array(\PHP_SAPI, ['cli', 'phpdbg', 'embed'], true) === false) {
 
 // Infection autoloading
 (static function (): void {
+    if (\file_exists($autoload = ($GLOBALS['_composer_autoload_path'] ?? ''))) {
```

Comment:
What do you think about checking if the global exists before checking if the file exists?

So we don't call file_exists on an empty string.